common/app/model/EmailAddons.scala

```diff
@@ -304,6 +304,11 @@ case object VoluntarySectorNetwork extends FrontEmailMetadata {
   override val banner = Some("voluntary-sector-network.png")
 }
 
+case object The Cityscape extends FrontEmailMetadata {
```

Comment:
We'll need to remove the space in `The Cityscape` here. This is what is causing the build to fail

Reply:
Sorry! Will make that change. Thanks very much